CHANGELOG.md

```diff
@@ -3,6 +3,63 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.15.0
+
+_Jul 29, 2022_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Introduce aggregation (#5592) @flaviendelangle
+
+  Premium users can now aggregate data in the grid.
+  Extract information like sum, average, count, and others with a couple of clicks.
+  
+  https://user-images.githubusercontent.com/45398769/181581503-77cc412e-9d9e-4de1-8bc3-c3bccc54cdaa.mp4
```

Comment:
I have opened https://github.com/orgs/community/discussions/27715 hoping that we will get a clean video support handling by GitHub in the future. 

I think that until then if the file size allows it, it might be better to have .gif. To be clear, I think that it's better to use videos in the blog, I'm considering the changelog here.

As a side note, I have found one way to get a clean gif for the first time on my environment. I have documented it in https://www.notion.so/mui-org/Blog-247ec2bff5fa46e799ef06a693c94917#502bd4e8933641ff8d660ef674150fba in case it can help for the future.